### ISSUE TITLE
NoSuchElementException during communication with CLI

### ISSUE THREAD

**Issue by dgelessus** [hhu-stups/prob-issues#61]
Copied from @favu100's comment: https://github.com/hhu-stups/prob-issues/issues/47#issuecomment-831212012

```
java.util.NoSuchElementException
at java.base/java.util.Scanner.throwFor(Scanner.java:937)
at java.base/java.util.Scanner.next(Scanner.java:1478)
at de.prob.cli.ProBConnection.getAnswer(ProBConnection.java:71)
at de.prob.cli.ProBConnection.send(ProBConnection.java:66)
at de.prob.cli.ProBInstance.send(ProBInstance.java:81)
at de.prob.animator.CommandProcessor.sendCommand(CommandProcessor.java:53)
at de.prob.animator.AnimatorImpl.execute(AnimatorImpl.java:60)
at de.prob.animator.ReusableAnimator.execute(ReusableAnimator.java:154)
at de.prob.animator.ReusableAnimator$InternalAnimator.execute(ReusableAnimator.java:43)
at de.prob.statespace.StateSpace.execute(StateSpace.java:577)
at de.prob.check.LTLChecker.execute(LTLChecker.java:46)
at de.prob.check.CheckerBase.call(CheckerBase.java:94)
at de.prob.check.LTLChecker.call(LTLChecker.java:12)
at de.prob2.ui.verifications.ltl.formula.LTLFormulaChecker.getResult(LTLFormulaChecker.java:123)
at de.prob2.ui.verifications.ltl.formula.LTLFormulaChecker.checkFormula(LTLFormulaChecker.java:81)
at de.prob2.ui.verifications.ltl.formula.LTLFormulaChecker.lambda$checkFormula$3(LTLFormulaChecker.java:101)
at java.base/java.lang.Thread.run(Thread.java:832)
```

I also encountered this issue before, but haven't been able to reproduce it. (It can also happen with other commands, not just LTL checking.) I think this is related to hhu-stups/prob2_kernel@f207b4f4c8df12a436e6ec569fba8dc64e89718f, which replaced some manual stream reading code with the standard Java `Scanner`. There's probably a small difference between our previous code and `Scanner` that causes this exception.

**Comment by dgelessus** [comment]
Looks like the same error as #39.

**Comment by dgelessus** [comment]
Fixed in hhu-stups/prob2_kernel@6e614cb35df12e00e3bd24c929aaf67ba7187753.
